pkg/util/resource.go

```diff
@@ -28,6 +28,10 @@ func NewZeroResourceList() corev1.ResourceList {
 	}
 }
 
+func NewResourceList() corev1.ResourceList {
```

Comment:
There is no need to abstract such a function

Reply:
done